lib/ClimaLandSimulations/experiments/README.md

```diff
@@ -9,22 +12,22 @@ julia> fluxnet_simulation("US-MOz")
 This will run ClimaLand with default parameters and domain. Optionally, you can modify these:
 
 ```jl
-julia> fluxnet_simulation("US-MOz"; customparams = path_to_parameters, customdomain = path_to_domain)
+julia> fluxnet_simulation("US-MOz"; site_setup_out = customsetup, params = customparams)
```

Comment:
what does "out" refer to in the kwarg?

Reply:
the variables could be `site_setup`, and the function could be called `make_site_setup` or `get_site_setup`. Same for the other `_out` variables - `domain_setup_out`, `site_config_out`, etc

Reply:
[can be later PR but I agree with Julia]

Reply:
I think I mentioned this at another point too, but "setup" is a little vague - what is it setting up? [later PR ok]